src/components/DashBoard.tsx

```diff
@@ -235,109 +241,263 @@ const Dashboard: React.FC = () => {
-          <Route path="/PageEditor" component={PageEditor} />
-          {isUserOfficer && <Route path="/CallPage" component={CallPage} />}
-          <Route path="/HelpPage" component={HelpPage} />
-          <Route path="/SEPPage/:id" component={SEPPage} />
-          <Route path="/SEPPage" component={SEPsPage} />
-          <Route path="/InstrumentPage" component={InstrumentsPage} />
-          <Route path="/InstitutionPage" component={InstitutionPage} />
-          <Route
+          <Page
+            setHeader={setHeader}
+            title="Proposal"
+            path="/ProposalPage"
+            component={ProposalPage}
+          />
+          <Page
+            setHeader={setHeader}
+            title="Page Editor"
+            path="/PageEditor"
+            component={PageEditor}
+          />
+          {isUserOfficer && (
+            <Page
+              setHeader={setHeader}
+              title="Call Page"
+              path="/CallPage"
+              component={CallPage}
+            />
+          )}
+          <Page
+            setHeader={setHeader}
+            title="Help Page"
+            path="/HelpPage"
+            component={HelpPage}
+          />
+          <Page
+            setHeader={setHeader}
+            title="SEP Page"
+            path="/SEPPage/:id"
+            component={SEPPage}
+          />
+          <Page
+            setHeader={setHeader}
+            title="SEPs Page"
+            path="/SEPPage"
+            component={SEPsPage}
+          />
+          <Page
+            setHeader={setHeader}
+            title="Instruments"
+            path="/InstrumentPage"
+            component={InstrumentsPage}
+          />
+          <Page
+            setHeader={setHeader}
+            title="Institution"
+            path="/InstitutionPage"
+            component={InstitutionPage}
+          />
+          <Page
+            setHeader={setHeader}
+            title="Merge Institution"
             path="/MergeInstitutionsPage/:institutionId"
             component={MergeInstitutionsPage}
           />
-          <Route
+          <Page
+            setHeader={setHeader}
+            title="Template Editor"
             path="/QuestionaryEditor/:templateId"
             component={TemplateEditor}
           />
-          <Route path="/ProposalTemplates" component={ProposalTemplatesPage} />
-          <Route
+          <Page
+            setHeader={setHeader}
+            title="Proposal Templates"
+            path="/ProposalTemplates"
+            component={ProposalTemplatesPage}
+          />
+          <Page
+            setHeader={setHeader}
+            title="Samples Templates"
             path="/SampleDeclarationTemplates"
             component={SampleTemplatesPage}
           />
-          <Route path="/GenericTemplates" component={GenericTemplatesPage} />
-          <Route
+          <Page
+            setHeader={setHeader}
+            title="Generic Template"
```

Comment:
There are a few inconsistencies in the naming of the pages like here "Samples Template**s**" vs "Generic Template" but other than that looks good.

Reply:
Thanks for today Tom. This has been resolved